### ISSUE TITLE
Cube files created by map2cam have the same serial number as the cube to match and can't be used together in qnet

### ISSUE THREAD

**Issue by ascbot** [DOI-USGS/ISIS3#1169]
---

Author Name: **Debbie Cook** (Debbie Cook)

---

When map2cam is used to create a level1 cube to use in control nets, the output level1 cube has the same serial number as the cube to match.  The goal of creating the level1 cube from the map is to select points for a control net, but this is not possible with duplicate serial numbers.

**Comment by ascbot** [comment]
---

Original Redmine Comment
Author Name: **Lynn Weller** (Lynn Weller)
Original Date: 2011-12-29T21:46:08Z

---

Example data available under /work/users/lweller/Isis3Tests/Cam2Map/

command line:
map2cam from=M10_Equi_1000meters_Clon180.cub match=EN0108821402M.cal.cub \
  to=M10_match_EN0108821402M.cub

**Comment by ascbot** [comment]
---

Original Redmine Comment
Author Name: **Stuart Sides** (@scsides)
Original Date: 2012-03-01T19:14:00Z

---

Type from major to bug. Priority normal to high.

**Comment by ascbot** [comment]
---

Original Redmine Comment
Author Name: **David Miller** (David Miller)
Original Date: 2015-07-08T22:38:00Z

---

Will consult with developers who work closely with map2cam, cnet, or qnet (and respond to announcement email) before starting work.

**Comment by ascbot** [comment]
I am a bot that cleans up old issues that do not have activity.

Happy Birthday to this issue! :birthday:

Unfortunately, this issue has not received much attention in the last 12 months. Therefore, I am going to close it. Please feel free to reopen this issue or open a new issue sometime in the future. If this issue is a bug report, please check that the issue still exists in our newest version before reopening.
